homeassistant/components/vicare/sensor.py

```diff
@@ -62,6 +62,10 @@
 SENSOR_GAS_CONSUMPTION_THIS_WEEK = "gas_consumption_heating_this_week"
 SENSOR_GAS_CONSUMPTION_THIS_MONTH = "gas_consumption_heating_this_month"
 SENSOR_GAS_CONSUMPTION_THIS_YEAR = "gas_consumption_heating_this_year"
+SENSOR_POWER_CONSUMPTION_TODAY = "power_consumption_heating_today"
+SENSOR_POWER_CONSUMPTION_THIS_WEEK = "power_consumption_heating_this_week"
+SENSOR_POWER_CONSUMPTION_THIS_MONTH = "power_consumption_heating_this_month"
+SENSOR_POWER_CONSUMPTION_THIS_YEAR = "power_consumption_heating_this_year"
```

Comment:
You have added these constants but the sensors are missing? Would be great to get power consumption sensors as well.

Reply:
THX for review.
I added the sonsors now.